app/views/candidate_mailer/submit_application_email.text.erb

```diff
@@ -23,4 +40,14 @@ Most training providers don’t reach a decision about your interview until they
 
 # Application decision
 
-Your training provider will respond to your application within 40 working days via email. If they decide to progress your application, they will communicate with you directly to organise an interview date and give you all the information you need.
+If a training provider decides to progress your application, they will communicate with you directly to organise an interview date and give you all the information you need.
+
+Your training provider will aim to make a decision on whether to make an offer within 40 working days of receiving your application.
+
+# Need help with your teacher training application?
+
+Call [Get into Teaching](https://getintoteaching.education.gov.uk/) on Freephone 0800 389 2500 or chat to an adviser using their [online chat service](https://getintoteaching.education.gov.uk/lp/live-chat) between 8am and 8pm, Monday to Friday.
```

Comment:
we might want not to hardcode `https://getintoteaching.education.gov.uk` and `0800 389 2500` into the email

Reply:
Why not?